src/OXRS_LCD.cpp

```diff
@@ -44,18 +47,64 @@ void OXRS_LCD::draw_logo(char * firmware_version)
   strcpy(buffer, "URC v");
   strcat(buffer, firmware_version);
   tft.drawString(buffer, 240/2, 40);
+**/
+  int logo_h = 0;
+  int logo_w = 0;
+  const unsigned char * logo_bm = NULL;
+  uint16_t logo_fg = TFT_BLACK;
+  uint16_t logo_bg = TFT_WHITE;
+  if (strcmp("SHA", fw_maker_code) == 0)
+  {
+    logo_h = sha40Height;
+    logo_w = sha40Width;
+    logo_bm = sha40;
+    logo_fg = tft.color565(0, 165, 179);
+    logo_bg = TFT_WHITE;
+  }
+  if (strcmp("AC", fw_maker_code) == 0)
+  {
+    logo_h = ac40Height;
+    logo_w = ac40Width;
+    logo_bm = ac40;
+    logo_fg = TFT_WHITE;
+    logo_bg = TFT_BROWN;
+  }
+  if (strcmp("BMD", fw_maker_code) == 0)
+  {
+    logo_h = bmd40Height;
+    logo_w = bmd40Width;
+    logo_bm = bmd40;
+    logo_fg = TFT_WHITE;
+    logo_bg = TFT_BLUE;
+  }
+ 
+  tft.drawBitmap(0, 0, logo_bm, logo_w, logo_h, logo_fg, logo_bg);
+  tft.fillRect(42, 0, 240, 40,  TFT_WHITE);
+  tft.setTextDatum( TL_DATUM);
+  tft.setTextColor(TFT_BLACK);
+  tft.setFreeFont(&Roboto_Light_13);
+  tft.drawString(fw_name, 46, 0);
+  tft.setFreeFont(&Roboto_Mono_Thin_13);
+  sprintf(buffer, "Maker: %s", fw_maker_code);
+  tft.drawString(buffer, 46, 13);
+  sprintf(buffer, "Vers.: %s / %s", fw_version, fw_platform); 
```

Comment:
I think `VER:` for the version header maybe? Capitals to match everything else?

Reply:
I would like to keep it here as is because it's in the header context with "Maker"

Reply:
Ok - can we drop the `.` then - so it is just `Vers:`? Would `Version:` fit?

Reply:
I changed the font for the whole header to proportional. `Version: `now fits and even that gained a little space